README.md

```diff
@@ -15,4 +16,23 @@ markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
 
+## Definition of little $o$ 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+## Definition of Big $O$
+$f(n)\in O(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) \le c g(n)$
+
+## Proof 
+- If we start with $f(n)\in o(g(n))$ by the deffiniton provided we know that for any positive constant $c$ there exists a point called 
+  $n_0$ beyond which is always $\ f(n) < c \cdot g(n)$
+- If we were to choose another constant $k$ that is greater then 0.
```

Comment:
So can we choose any other constant here?

Reply:
Yes, k can be be any constant that is greater then 0. As long as f(n) < k*g(n) it implies that f(n) <= k*g(n)

Reply:
Why can we choose any constant here?

Reply:
This is because the definition of little o allows us to. It states that for all constants there exists a point n_0 beyond which f(n) < cg(n). No matter how much we scale g(n), f(n) will at some point (n_0) be smaller then cg(n). It is essentially saying the f(n) grows much slower then g(n) as n increases.

Reply:
Can you make that explicit in your proof please?